app/frontend/src/features/profile/EditHostingPreferencePage.tsx

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+import PageTitle from "../../components/PageTitle";
+import HostingPreferenceForm from "./HostingPreferenceForm";
+import { theme } from "../../theme";
+
+const useStyles = makeStyles({
+  title: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
+});
+
+function EditHostingPreferencePage() {
+  const classes = useStyles();
+
+  return (
+    <>
+      <PageTitle className={classes.title}>Edit my place</PageTitle>
```

Comment:
Why does this title need extra styling? What makes it different from the title of EditProfilePage?

Reply:
This title is too close together to the "Hosting preferences" title immediately below it, which isn't a problem in the edit profile page. Probably solve this differently by having margin top on the form instead, don't have a strong preference on this one either way. What do you think?

Reply:
Maybe all PageTitles need that space around them.

Reply:
That's also why I prefer someone to take ownership of the styling. @navandstokes maybe?

Reply:
I think for now while we're still in early stages, just make the change for all component styles if you think it's suitable

Reply:
Should we set some default rules? I think by default vertical spacing should be done by margin top. Should we also be using `theme.spacing()` everywhere?

Reply:
I think we should use `theme.spacing()` over random values whenever we can. It keeps our spacing across the app more consistent that way.